tests/executors/test_kubernetes_executor.py

```diff
@@ -205,7 +205,7 @@ def test_run_next_exception(self, mock_get_kube_client, mock_kubernetes_job_watc
         try_number = 1
         kubernetes_executor.execute_async(key=('dag', 'task', datetime.utcnow(), try_number),
                                           queue=None,
-                                          command='command',
+                                          command=["airflow", "version"],
```

Comment:
This is still an invalid command. Or is the change really just to make it a list?

Reply:
Updated in https://github.com/apache/airflow/pull/9240/commits/3c61dfd52781231d567a4859a72d1667e1491765